packages/shell-api/src/field-level-encryption.ts

```diff
@@ -75,17 +82,17 @@ export class ClientEncryption extends ShellApiClass {
       value,
       {
         keyId: encryptionId,
-        algorithm: encryptionAlgorithm
```

Comment:
If you wanted you could make the type of encryptionAlgorithm more specific to match the ClientEncryption types, to avoid any, but it's not a big deal